botbot/apps/plugins/core/logger.py

```diff
@@ -26,12 +34,18 @@ def logit(self, line):
         if line._channel_name.startswith("#"):
             ignore_prefix = self.config['ignore_prefix']
             
+            if ignore_prefix:
+                if not isinstance(ignore_prefix, list):
+                    ignore_prefix = [ignore_prefix]
```

Comment:
self.config['ignored_prefix'] will return the default Config option, which in this case is "!-" (now changed to ["!-"]. However, the user could set the config to {ignore_prefixes: None} (I think), which is one reason why we're checking for it here. The other reason was checking for empty strings - I'll add that back in...